### PR TITLE
Release 0.84.0

### LINKED ISSUE #2071: Release the operator v0.84.0

### PR DESCRIPTION
**Pull request by yuriolisa**
Resolves #2071

### CODE REVIEW

**Review by swiatekm-sumo — APPROVED** [review]

**Review by iblancasa — APPROVED** [review]

**Comment by TylerHelmuth** [comment]
Would be great to get these 2 PRs merged first: 
- https://github.com/open-telemetry/opentelemetry-operator/pull/2087
- https://github.com/open-telemetry/opentelemetry-operator/pull/2096

**Comment by TylerHelmuth** [comment]
https://github.com/open-telemetry/opentelemetry-operator/pull/1683 was out-of-date, I have closed it.

File: versions.txt
Comment (Kielek) [inline versions.txt:1]: Please update also .NET version here as #2096 was merged

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,23 @@ Changes by Version
...
+- `operator`: Limit auto-instrumentation emptydir volume size (#2044)
+- `operator`: Make OpenShift routes work with missing hostname (#2074)
+  If the Ingress hostname is not specified OpenShift route hostname is set to `<port-name>-<otel-cr-name>-route-<otel-cr-namespace>-basedomain`.
+
+### 🧰 Bug fixes 🧰
+
+- `operator`: Avoid running the auto-instrumentation pod mutator for pods already auto-instrumented (#1366)
+- `autoinstrumentation`: Allow the usage of the Apache HTTPD autoinstrumentation to be run as non-root user. Change the files permission to allow their copy from a non-root user. (#2068)
+- `operator`: Fixes reconciling otel-collector service's internal traffic policy changes. (#2061)
+- `operator`: Make OpenShift Route work with gRPC receivers by using h2c appProtocol (#1969)
```
Comment (frzifus) [inline CHANGELOG.md:21]:
```suggestion
- `operator`: Make OpenShift Route work with gRPC receivers by using h2c appProtocol (#1969)

### Components
TBD
```

Should we add the component section here too?

**Review by frzifus — CHANGES_REQUESTED** [review]
LGTM, only one tiny thing ^^

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,35 @@
 Changes by Version
 ==================
+## 0.84.0
 
-<!-- next version -->
```
Comment (pavolloffay) [inline CHANGELOG.md:4]: This should not be removed and instead moved  above the #0.84.0

**Review by frzifus — APPROVED** [review]

**Comment by pavolloffay** [comment]
The PR needs to be rebased to include https://github.com/open-telemetry/opentelemetry-operator/blob/main/.chloggen/remove_ta_defaults.yaml

**Review by pavolloffay — APPROVED** [review]

**Pull request merged**